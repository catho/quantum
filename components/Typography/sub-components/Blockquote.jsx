import classNames from 'classnames';
import styles from './Blockquote.module.css';

const Blockquote = ({ className = '', children, ...props }) => {
  const classContent = classNames(styles.blockquote, className);
  return (
    <blockquote className={classContent} {...props}>
      {children}
    </blockquote>
  );
};

export default Blockquote;
