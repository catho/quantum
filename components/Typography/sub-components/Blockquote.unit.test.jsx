import { render } from '@testing-library/react';

import Blockquote from './Blockquote';

describe('<Blockquote />', () => {
  it('Should match the snapshot', () => {
    expect(render(<Blockquote />).asFragment()).toMatchSnapshot();
  });
});
