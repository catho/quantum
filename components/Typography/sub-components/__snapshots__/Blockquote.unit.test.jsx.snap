// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blockquote /> Should match the snapshot 1`] = `
<DocumentFragment>
  .Blockquote-module__blockquote___Xqdgz {
  font-size: 12px;
  letter-spacing: .2px;
  line-height: 18px;
  margin: 0;
}

<blockquote
    class="Blockquote-module__blockquote___Xqdgz"
  />
</DocumentFragment>
`;
